packages/@adobe/spectrum-css-temp/components/table/index.css

```
@@ -34,6 +34,11 @@ governing permissions and limitations under the License.
   transition: transform var(--spectrum-global-animation-duration-100) ease-in-out;
 }
 
+.spectrum-Table-head {
+  border-left: 1px solid transparent;
+  border-right: 1px solid transparent;
```

```suggestion
  border-left-width: 1px;
  border-left-style: solid;
  border-right-width: 1px;
  border-right-style: solid;
```
`border-*-color: transparent` should be added to skin.css

using a shorthand that includes color has bitten us in other components, i forget exactly how

---

I moved this into the JSX and out of the css files. This was because the div with role="presentation" had to className and the :parent selector wasn't working. I'm wondering if I should follow the above and five the div a className.